data/provider/src/test/java/tech/pegasys/teku/api/ChainDataProviderTest.java

```diff
@@ -548,6 +549,33 @@ public void calculateRewards_shouldGetData() {
         .containsExactlyInAnyOrder(Map.entry(2, -1 * reward), Map.entry(4, reward));
   }
 
+  @Test
+  public void calculateSyncAggregateBlockRewards_manySyncAggregateIndices() {
+    final UInt64 reward = UInt64.valueOf(1234);
+    final Spec spec = TestSpecFactory.createMinimalAltair();
+    final DataStructureUtil data = new DataStructureUtil(spec);
+    final ChainDataProvider provider = setupAltairState();
+    final SyncAggregate syncAggregate =
+        data.randomSyncAggregate(0, 3, 4, 7, 16, 17, 20, 23, 25, 26, 29, 30);
+
+    final int syncAggregateBlockRewards =
+        provider.calculateSyncAggregateBlockRewards(reward, syncAggregate);
+    assertThat(syncAggregateBlockRewards).isEqualTo(reward.times(12).intValue());
```

Comment:
Where does the value 12 come from? Is it the number of participant validators? It is probably worth assigning it to a variable with a name explaining the rationale to make the test more readable.

Another option would be something like this:
```
int[] participantsValidatorIndexes = new int[] {0, 3, 4, 7, 16, 17, 20, 23, 25, 26, 29, 30};
final SyncAggregate syncAggregate = data.randomSyncAggregate(participantsValidatorIndexes);

[...]

assertThat(syncAggregateBlockRewards).isEqualTo(reward.times(participantsValidatorIndexes.length).intValue());
```